spec/features/books/new_spec.rb

```diff
@@ -0,0 +1,167 @@
+require "rails_helper"
+
+RSpec.describe "As a visitor" do
+  describe "when I visit the book index page" do
+    it "has a link to take me to the new book path" do
+      visit books_path
+
+      click_link("Add new book")
+
+      expect(page.status_code).to eq(200)
+      expect(current_path).to eq(new_book_path)
+    end
+  end
+
+  describe "when I am at the new book form" do
+    before :each do
+      @title = "The Illiad"
+      @year_published = 850
+      @page_count = 443
+      @thumbnail = "https://images.gr-assets.com/books/1388188509l/1371.jpg"
+      @authors = "Homer"
+    end
+
+    it "allows users to input information and create a book" do
+      visit new_book_path
+
+      fill_in :book_title, with: @title
+      fill_in :book_year_published, with: @year_published
+      fill_in :book_page_count, with: @page_count
+      fill_in :book_thumbnail, with: @thumbnail
+      fill_in :book_authors, with: @authors
+
+      click_button "Create Book"
+
+      new_book = Book.last
+      expect(current_path).to eq(book_path(new_book))
```

Comment:
Add RSpec description (or make this a separate `it` statement) that the page redirects to the book show page after successful book creation.

Reply:
Separated into a new test.